bin/infrasim-init

```diff
@@ -54,20 +55,21 @@ def create_infrasim_directories():
         os.system("rm -rf /usr/local/libexec")
         os.mkdir("/usr/local/libexec")
 
+
 def create_infrasim_conf():
 
     # Prepare default network
     networks = []
     nics_list = netifaces.interfaces()
-    eth_nic = filter(lambda x: 'e' in x,nics_list)[0]
+    eth_nic = filter(lambda x: 'e' in x, nics_list)[0]
```

Comment:
Write it in another way

    def is_eth_or_ens(x):
        if 'e' in x:
            return True
        else:
            return False

    nics_list = [a, b, c, d]  # already got by some functions
    list_ethernets = []
    for nic in nics_list:
        if is_eth_or_ens(nic):
            list_ethernets.append(nic)
    eth_nic = list_ethernets[0]

lambda provides a quick function, while filter() picks up all elements if function returns True and finally compose a list.

This line means pick up one nic from all nics_list and the nic must have "e".